styles/index.css

```diff
@@ -0,0 +1,121 @@
+.main-box {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 1500px;
+    height: 1000px;
+    padding: 10px;
+    margin: 20px;
+    background-color: green;
+}
+
+#temp-box {
```

Comment:
I would look in to this resource https://css-tricks.com/snippets/css/a-guide-to-flexbox/ to help you in laying out your child elements.